### ISSUE TITLE
Unable to resolve module react-dom from xx/node_modules/dva/lib/createDva.js

### ISSUE THREAD

**Issue by buhe** [sorrycc/dva-example-react-native#6]
Found
```
Unable to resolve module react-dom from xx/node_modules/dva/lib/createDva.js
```
in
"dva": "^1.2.1",
"react": "15.4.2",
"react-native": "0.40.0",

```
function render(container, store, app, router) {
      const ReactDOM = require('react-dom');
      ReactDOM.render(React.createElement(getProvider(store, app, router)), container);
    }
```
React Native env need react-dom? How to solve it?

Thanks.

**Comment by sorrycc** [comment]
Yes, it's a problem with dva. We may resolve it in next version. 

> React Native env need react-dom?

No. dva depend on it, but it will not be used in project with react-native.

> How to solve it?

Install react-dom dependency.

**Comment by feit** [comment]
after install react-dom, this example also can not run

**Comment by ouzhou** [comment]
something goes to bad
![image](https://cloud.githubusercontent.com/assets/7693866/24826887/e3710d30-1c72-11e7-99b5-2508113f2dd2.png)
this is my dependencies
![image](https://cloud.githubusercontent.com/assets/7693866/24826890/f7354a2a-1c72-11e7-911b-ef5cd3285c69.png)
so how to solve it?

**Comment by tmzhenzheng** [comment]
请问dva 这个问题解决了么
